package.json

```diff
@@ -1,8 +1,10 @@
 {
   "scripts": {
     "lint-scss": "sass-lint static/**/*.scss --verbose --no-exit",
-    "build": "mkdir -p static/js && cp node_modules/global-nav/dist/index.js static/js/global-nav.js && cp node_modules/cookie-policy/build/js/cookie-policy.js static/js/cookie-policy.js && yarn run build-css",
-    "build-css": "node-sass --include-path node_modules static/sass --output static/css && postcss --use autoprefixer --replace 'static/css/**/*.css' && postcss --use cssnano --dir static/minified 'static/css/**/*.css' && cp node_modules/cookie-policy/build/css/cookie-policy.css static/css/cookie-policy.css",
+    "build": "mkdir -p static/js && yarn run build-cookie-policy && yarn run build-global-nav && yarn run build-css",
+    "build-cookie-policy": "cp node_modules/@canonical/global-nav/dist/index.js static/js/global-nav.js",
```

Comment:
I think you should have one `build-js` one, which is what we do on other projects.

But if you think that the command is going to be inherently too long, I would suggest to extract those commands into a `scripts/build-js` script:

``` bash
#! /usr/bin/env bash

cp node_modules/@canonical/global-nav/dist/index.js static/js/global-nav.js
cp node_modules/@canonical/cookie-policy/build/js/cookie-policy.js static/js/cookie-policy.js
```

Then make it executable (`chmod +x scripts/build-js`) and do:

```
"build-js": "./scripts/build-js"
```